src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -87,6 +87,22 @@ export const QueryResultPane = () => {
 			renderHeaderCell: () => <>{MESSAGE}</>
 		}),
 	];
+	const gridParentRef = useRef<HTMLDivElement>(null);
+	const ribbonRef = useRef<HTMLDivElement>(null);
+	// Resize grid when parent element resizes
+	useEffect(() => {
+		const gridParent = gridParentRef.current;
+		if (!gridParent) {return;}
+        const observer = new ResizeObserver(() => {
+			if (!gridRef.current) {return;}
+			if (!ribbonRef.current) {return;}
+			if (gridParent.clientWidth && gridParent.clientHeight) {
+				gridRef.current.resizeGrid(gridParent.clientWidth, gridParent.clientHeight - ribbonRef.current.clientHeight - 5);
```

Comment:
How does this work when we have multiple grids?

Reply:
We haven't added functionality for multiple result sets yet, do you think this will cause issues when we implement that?